### DISCUSSION TITLE
PHP GMP & intl

### DISCUSSION THREAD

**Discussion by donniewr** [andreapollastri/cipi#396] [Ideas]
Hello,

Cipi should install these PHP extensions by default for all PHP versions, I have used CIPI with many PHP projects and GMP and intl was always recommended.

Thanks,

**Comment by andreapollastri** [comment]
Hello guys, it the last 3 years the world has been changed with AI and with new deployments needs so a new "Laravel Control Panel" is not the real solution for today's world. I have created a new stable version of Cipi full of features and based on a powerful and light CLI. If you would like to try it, here you can find all informations: https://cipi.sh/
